### PR TITLE
GQL - Add Broadcast & FollowerCount Query to User

### PR DESCRIPTION
**Pull request by AnatoleAM**
Adding two new fields to the `User` GQL type

- Broadcast: the user's active live broadcast, if any
- Follower Count: the user's current amount of followers

This is used for the User Page on the Web App.

### CODE REVIEW

**Comment by TroyDota** [comment]
For what purpose?

**Comment by TroyDota** [comment]
And do we need all the fields in the broadcaster

File: src/server/api/v2/gql/resolvers/query/user.go [hunk 1]
```diff
@@ -499,3 +499,41 @@ func (r *UserResolver) AuditEntries() (*[]*auditResolver, error) {
...
+		return 0
+	}
+
+	return count
+}
+
+// Get user's live broadcast, if any
+func (r *UserResolver) Broadcast() (*datastructure.Broadcast, error) {
+	var stream *datastructure.Broadcast
+	if streams, err := api_proxy.GetTwitchStreams(r.ctx, r.v.Login); err != nil {
```
Comment (TroyDota) [inline src/server/api/v2/gql/resolvers/query/user.go:516]: Stream?

**Comment by AnatoleAM** [comment]
> For what purpose?

Displaying more contextual information on 7tv.app user pages 

> And do we need all the fields in the broadcaster

Some of them likely won't be used but since it's public information provided by twitch there's no harm in allowing them to be queried

File: src/server/api/v2/gql/resolvers/query/user.go (hunk 1 above)
Comment (AnatoleAM) [reply src/server/api/v2/gql/resolvers/query/user.go:516]: The initial response from twitch is a list of streams, so it's `streams`.

**Review by TroyDota — APPROVED** [review]

**Pull request merged**